rest_framework/authtoken/management/commands/drf_create_token.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from rest_framework.authtoken.models import Token
+
+
+class Command(BaseCommand):
+    help = 'Create DRF Token for a given user'
+
+    def create_user_token(self, username):
+        user = User.objects.get(username=username)
```

Comment:
@jpadilla I will take this advice into consideration, thanks! I'm learning a lot of things :)